### PR TITLE
Added event buffer documentation for version 1.0.0

### PR DESCRIPTION
**Pull request by jpatel531**
Adding documentation a trigger section for more specific detail about socket_id / event buffer

### CODE REVIEW

**Comment by jpatel531** [comment]
Same format as before : https://github.com/pusher/pusher-http-python/tree/event_buffer_documentation

File: README.md [hunk 1]
```diff
@@ -51,17 +51,41 @@ contain alphanumeric characters, `-` and `_`:
...
-If you wish to trigger an event on multiple channels, pass in a list as the first parameter:
+Triggering Events
+-----------------
 
-```python
-pusher.trigger([u'a_channel', u'another_channel'], u'an_event', {u'some': u'data'})
-```
+To trigger an event on one or more channels, use the `trigger` method on the `Pusher` object.
+
+##### Event Buffer
```
Comment (leggetter) [inline README.md]: How do I for each section first then details. We're going into the Event Buffer stuff too soon here. See:

* https://github.com/pusher/pusher-http-php/tree/event-buffer#event-buffer
* https://github.com/pusher/pusher-http-dotnet/tree/event-buffer#event-buffer

**Comment by jpatel531** [comment]
Good point! Will amend

**Comment by jpatel531** [comment]
Moved Event Buffer to after the API guide. WDYT @leggetter ?

**Pull request merged**